src/js/cloud-animation.js

```diff
@@ -0,0 +1,10 @@
+let scrollPos = 5;
+const cloudSection = document.querySelector(".c-main-section__cloud")
+function checkPosition() {
+  let windowY = window.scrollY;
+  if (windowY > scrollPos) {
+    cloudSection.classList.add("c-main-section__cloud-animation-right");
+  }
+  scrollPos = windowY;
+}
+document.addEventListener('scroll', checkPosition);
```

Comment:
so proud of this js magic ! -![](https://media.giphy.com/media/11wsZr0jbXc15m/giphy.gif)